api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/specialClinics/SpecialClinicsReferralStatusDataEvaluator.java

```diff
@@ -42,7 +42,7 @@ public EvaluatedEncounterData evaluate(EncounterDataDefinition definition, Evalu
         String qry = "select v.encounter_id,\n" +
                 "(case v.referral_status when 5485 then 'Refer for admission' when 164407 then 'Referral for other clinics' when 163316 then 'Referral for livelihood support' when 159492 then 'Transferred Out' else '' end) as referral_status\n" +
                 "from kenyaemr_etl.etl_special_clinics v\n" +
-                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic = :specialClinic;";
+                "where date(v.visit_date) between date(:startDate) and date(:endDate) and special_clinic_form_uuid = :specialClinic;";
```

Comment:
Special clinic form uuid is a String. Ensure the parameter value is in quotes, preferrably single quotes.